### ISSUE TITLE
Record formatting applied to expressions with RecordWildCards

### ISSUE THREAD

**Issue by ttuegel** [commercialhaskell/hindent#274]
In the following example,

``` .haskell
foo (Bar {..}) = 
    Bar {..}
```

hindent wants the record wild card on the right-hand side of `foo` to span multiple lines like so:

``` .haskell
foo (Bar {..}) = 
    Bar
    { ..
    }
```

**Comment by chreekat** [comment]
I feel record wild cards should be special cased to be part of the word that precedes them. So `Bar{..}` rather than the other alternatives. That's the only form I've ever seen used, but maybe I haven't looked too hard. :)

**Comment by adamse** [comment]
@chrisdone this appears to be fixed, the example now prettifies to
```
foo (Bar {..}) = Bar {..}
```

**Comment by sighingnow** [comment]
Close as fixed in version 5.2.2. @ttuegel feel free to reopen this thread if you have any further question about the issue.
